static/js/containers/PostDetailSidebar_test.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import R from "ramda"
+import sinon from "sinon"
+import { assert } from "chai"
+
+import { actions } from "../actions"
+import PostDetailSidebar, {
+  PostDetailSidebar as InnerPostDetailSidebar
+} from "./PostDetailSidebar"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { makePost } from "../factories/posts"
+import { makePostResult } from "../factories/search"
+
+describe("PostDetailSidebar", () => {
+  let helper, render, post, relatedPostData
+
+  beforeEach(() => {
+    post = makePost()
+    relatedPostData = R.times(makePostResult, 4)
+    helper = new IntegrationTestHelper()
+    helper.getRelatedPostsStub.returns(Promise.resolve(relatedPostData))
```

Comment:
@aliceriot This is the test file that I mentioned earlier. The `PostDetailSidebar` component executes a `post` on the `relatedPosts` redux hammock endpoint. In the app, the `postSuccessHandler` is invoked, but for whatever reason I cannot get it to run in these tests. As a result I have to stub this API method to return the value that `postSuccessHandler` is actually returning. Obviously I'd prefer not to do that, so if you have any ideas about what might be going wrong, let me know